assets/global.js

```diff
@@ -1124,6 +1124,7 @@ class VariantSelects extends HTMLElement {
 
         if (pricePerItemSource && pricePerItemDestination) {
           pricePerItemDestination.innerHTML = pricePerItemSource.innerHTML;
+          pricePerItemDestination.classList.toggle('visibility-hidden', pricePerItemSource.classList.contains('visibility-hidden'));
```

Comment:
Could we replace the entire element instead of replacing its innerHTML and toggling its class in 2 separate steps?
```suggestion
          pricePerItemDestination.replaceWith(pricePerItemSource);
```